src/test/java/net/lingala/zip4j/ExtractZipFileIT.java

```diff
@@ -301,6 +303,24 @@ public void testExtractZipFileWithZip64ExtraDataRecordWithOnlyFileSizesInIt() th
     assertThat(Files.walk(outputFolder.toPath()).filter(Files::isRegularFile)).hasSize(1);
   }
 
+  /**
+   * this test file is got from issue#45
+   * @throws IOException
+   */
```

Comment:
Please remove this javadoc. If needed we can look at git log to see what this change was made for.